subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishCoordinatesIntegTest.groovy

```diff
@@ -143,4 +143,120 @@ class MavenPublishCoordinatesIntegTest extends AbstractMavenPublishIntegTest {
         }
     }
 
+    def "fails when multiple publications share the same coordinates"() {
```

Comment:
There's essentially the same test (ignored) at https://github.com/gradle/gradle/blob/29923e47387b62daf320c6434d127949b906c25d/subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishBasicIntegTest.groovy#L280.  We should remove it.